test/nullable.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+import Base.Dates: Millisecond
```

Comment:
Since it's used only once and isn't related to nullables in general, better write `Base.Dates.Milliseconds` once below instead of importing.